native/src/main/java/bblfsh/bash/Main.java

```diff
@@ -0,0 +1,18 @@
+package bblfsh.bash;
+
+public class Main {
+
+    public static void main(String args[]) {
+        final RequestReader reader = new RequestReader(System.in);
```

Comment:
I think it is ok like it is, but maybe, to do not have the Main class in tests, you could create a Singleton to provide the mapper.